tests/factories.py

```diff
@@ -176,6 +189,12 @@ def create_org(self, **kwargs):
 
         return org
 
+    def create_table_metadata(self, **kwargs):
+        return table_metadata_factory.create(**kwargs)
+
+    def create_column_metadata(self, **kwargs):
+        return column_metadata_factory.create(**kwargs)
+
```

Comment:
Here's a workaround that makes the tests pass:

```diff
@@ -286,7 +286,10 @@ class Factory(object):
             'org': self.org
         }
         args.update(kwargs)
-        return query_factory.create(**args)
+        q = query_factory.create(**args)
+        if 'schedule_failures' in args:
+            q.schedule_failures = args['schedule_failures']
+        return q
 
     def create_query_with_params(self, **kwargs):
         args = {
```

Reply:
Ok, slapped it on 0b12ae3da9570412da3d9cf07f6c36f85ad93d24